core/src/main/java/com/linecorp/armeria/client/retry/RetryStrategy.java

```diff
@@ -113,10 +114,31 @@
      * If the condition does not match, this will return {@code null} to stop retry attempt.
      * Note that {@link ResponseTimeoutException} is not retriable for the whole retry, but each attempt.
      *
+     * @param ctx the {@link ClientRequestContext} of this request
+     * @param response the {@link Response} from the server.
+     *                 If {{@link #needsResponseStreaming(ClientRequestContext)}} returns {@code false} and
+     *                 the type of {@link Response} is {@link HttpResponse}, this will be {@code null}.
```

Comment:
How about:

```
{@code null} if {@link #needsResponseStreaming(...)} ....
The actual {@link Response} otherwise.
```

Please also note the double curly braces in `{{@link #needsResponseStreaming...}}`